extensions/ch_vd_sire/views/zone_pression.sql

```diff
@@ -0,0 +1,12 @@
+CREATE VIEW qwat_ch_vd_sire.zone_pression AS
+	SELECT
+		pressurezone.id AS "ID_Num",
+		NULL::text AS "Remarque",
+		pressurezone.name AS "Nom_Descriptif",
+		distributor.name AS "Nom_Distributeur",
+		consumptionzone.qwat_ext_ch_vd_sire_pressure_load_type AS "Mise_En_Charge",
+		consumptionzone.qwat_ext_ch_vd_sire_pressure_load_altitude AS "Altitude_Plan_De_Charge"
+		FROM qwat_od.pressurezone
+		JOIN qwat_od.distributor ON pressurezone.fk_distributor = distributor.id
+		JOIN qwat_od.consumptionzone ON pressurezone.fk_consumptionzone = consumptionzone.id
+	WHERE pressurezone.fk_distributor = 1;
```

Comment:
+GRANT SELECT, REFERENCES, TRIGGER ON TABLE qwat_ch_vd_sire.captage TO qwat_viewer;
+GRANT ALL ON TABLE qwat_ch_vd_sire.captage TO qwat_user;
+GRANT ALL ON TABLE qwat_ch_vd_sire.captage TO qwat_manager;

(add Grants)